core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -104,6 +104,11 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     consumer.subscribe(tp)
     consumer.seek(tp, 0)
 
+    // wait until all the followers have synced the last HW with leader
```

Comment:
Do you think it's worthwhile to mention the JIRA for the issue that this check gets around (i.e. KAFKA-1211)?